### PR TITLE
Add navigation sidebar from draft mockup

### PR DESCRIPTION
**Pull request by kalvinwang**
Adds an unstyled version of the sidebar from the draft mockup in  https://github.com/cagov/benefits-screener/issues/21:

image

GIF of the sidebar in its current state:

sidebar nav

Ran tests, updated snapshots

### CODE REVIEW

File: src/client/Sidebar.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import React from "react";
+import Row from "react-bootstrap/Row";
+import Tab from "react-bootstrap/Tab";
+import { useTranslation } from "react-i18next";
+
+function Sidebar() {
+  const { t } = useTranslation();
+
+  // We write these out instead of generating them
+  // so we can find them easily when searching for keys in translation.json
```
Comment (sharonwarner) [inline src/client/Sidebar.js:12]: 👍

File: src/client/Sidebar.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import Col from "react-bootstrap/Col";
+import Nav from "react-bootstrap/Nav";
+import React from "react";
+import Row from "react-bootstrap/Row";
+import Tab from "react-bootstrap/Tab";
+import { useTranslation } from "react-i18next";
+
+function Sidebar() {
```
Comment (sharonwarner) [inline src/client/Sidebar.js:8]: Could we add a snapshot test for this component? In the past, we've used the file structure: `client/components/*component_name*/index.js`
`client/components/*component_name*/index.test.js`
`client/components/*component_name*/__snapshots__` (files are added to this folder automatically)

I thought that file structure worked well, but I'm definitely open to other ideas too!

Comment (kalvinwang) [reply src/client/Sidebar.js:8]: ah right, sure!

**Comment by kalvinwang** [comment]
Forgot to tag you @idealisms, I'm planning to do that on future PRs but not as a blocker, you're welcome to review if you have the time / interest. Thanks!

File: src/client/Sidebar.js (hunk 2 above)
Comment (kalvinwang) [reply src/client/Sidebar.js]: added!

**Comment by idealisms** [comment]
LGTM2

**Pull request merged**